src/main.zig

```diff
@@ -271,6 +272,7 @@ fn fetch(arena: Allocator, args: []const []const u8) !void {
         11 => .big_sur,
         12 => .monterey,
         13 => .ventura,
+        14 => .sonoma,
```

Comment:
If you grep for `.ventura` you'll see you missed a couple more spots in the code.